cfgov/v1/management/commands/delete_page.py

```diff
@@ -0,0 +1,40 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from v1.models import CFGOVPage
+
+
+class Command(BaseCommand):
+    help = 'Deletes a page by its slug name'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--slug',
+            nargs='+',
+            help='The slug for the page you wish to delete'
+        )
+        parser.add_argument(
+            '--id',
+            nargs='+',
+            help='The ID for the page you wish to delete'
+        )
+
+    def handle(self, *args, **options):
+        page_to_delete = None
+        if options['slug']:
+            slug = options['slug'][0]
+            try:
+                page_to_delete = CFGOVPage.objects.get(slug=slug)
+            except Exception as e:
```

Comment:
I would probably prefer to remove the ability to delete by slug because of non-uniqueness, but if you want to leave it in, can you catch [`MultipleObjectsReturned`](https://docs.djangoproject.com/en/1.8/ref/exceptions/#multipleobjectsreturned) here instead of the generic `Exception`, to avoid hiding some other error from being hidden (e.g. the database connection fails)?